src/back/glsl.rs

```diff
@@ -899,7 +899,6 @@ fn write_storage_class(class: StorageClass) -> Result<String, Error> {
 
 fn write_interpolation(interpolation: Interpolation) -> Result<&'static str, Error> {
     Ok(match interpolation {
-        Interpolation::Perspective => "smooth",
```

Comment:
hold on, let's not rush into removing this one, ok? there is no harm to keep an explicit variant, in case we want to make it required later